xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -305,7 +392,7 @@ CGDisplayModeRef BestMatchForMode(CGDirectDisplayID display,
   // Try to find a mode with the requested depth and equal or greater dimensions first.
   // If no match is found, try to find a mode with greater depth and same or greater dimensions.
   // If still no match is found, just use the current mode.
-  CFArrayRef allModes = GetAllDisplayModes(display);
+  CFArrayRef allModes = CopyAllDisplayModes(display);
```

Comment:
not related to your PR, but now reading through this BestMatchForMode function I see that it returns non-retained value which I don't really like. Could you change that? Here's a similar example: https://github.com/p00ya/displaymode/blob/master/displaymode.c#L264